core/vnl/vnl_matlab_print_scalar.h

```diff
@@ -32,6 +33,34 @@ vnl_matlab_print_scalar_declare(std::complex<float>);
 vnl_matlab_print_scalar_declare(std::complex<double>);
 vnl_matlab_print_scalar_declare(std::complex<long double>);
 
+
+
+
+
+
+//: print real or complex scalar into character buffer.
+#define vnl_matlab_print_scalar_declare_old(T) \
+VNL_EXPORT void vnl_matlab_print_scalar(T v, \
```

Comment:
I think this could have `VXL_DEPRECATED` too, ie:

`VNL_EXPORT VXL_DEPRECATED void vnl_matlab_print_scalar(T v, \`

Worth a try anyway.